production/tanka/grafana-agent/v1/internal/kubernetes_logs.libsonnet

```diff
@@ -1,185 +1,7 @@
-local gen_scrape_config(job_name, pod_uid) = {
```

Comment:
Heads up this looks like it got changed from gen_scrape_config -> gen_logs_scrape_config which broke the bash script.